README.md

```diff
@@ -118,6 +118,13 @@ The only required input is `project-name`.
 1. **disable-github-env-vars** (optional) :
    Set to `true` if you want do disable github environment variables in codebuild.
 
+1. **stop-on-signals** (optional) :
```

Comment:
I was trying to test this PR and could not able to cancel the workflow and the build .

my workflow: https://github.com/taoyong-ty/aws-codebuild-run-build/actions/runs/7731467637/workflow 

```
name: Run CodeBuild Action

on: push

jobs:
  codebuild-job:
    runs-on: ubuntu-latest
    name: CodeBuild Job
    steps:
      - name: Configure AWS Credentials
        uses: aws-actions/configure-aws-credentials@v1
        with:
          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
          aws-region: us-west-2
      - name: Run CodeBuild
        id: codebuild
        if: always()
        uses: freckle/aws-codebuild-run-build@0881951f31dfd749aea98e4dcb1bcc7d8f9dbc05
        with:
          project-name: aws-codebuild-run-build
          buildspec-override:   |
            version: 0.2
            phases:
              build:
                commands:
                  - sleep 300
          hide-cloudwatch-logs: false
          artifacts-type-override: NO_ARTIFACTS
```

Reply:
Can you say a little more about "couldn't cancel the workflow and the build"? The logs seem to have expired for that example. If you do it again, I can take a look.

This has been working well for us since I opened this PR. Here is a recent example,

GitHub cancels the build (it was a `fail-fast` matrix Job where another Job failed):

![](https://files.pbrisbin.com/screenshots/screenshot.1949324.png)

You can see the Build did indeed go into Stopped at exactly that time:

![](https://files.pbrisbin.com/screenshots/screenshot.1949361.png)

And the end of the logs on the CodeBuild side, showing the same:

![](https://files.pbrisbin.com/screenshots/screenshot.1949385.png)

Reply:
I don't now why you have `if: always()` there, can you try removing it? `always()` is an anti-pattern anyway because it includes the `cancelled` status too, which feels important here.

Reply:
you are right. 

What I saw earlier was that the CodeBuild build was still running, even though the GitHub has acknowledged the cancelling request.

But I am now able to see the expected behavior after removin `if: always()`!

Reply:
Thanks for making the change!

I am ready to approve and merge the change. Could you rebase your commit?

Reply:
Will do!

Reply:
OK, done.